mkdocs.yml

```diff
@@ -139,6 +139,7 @@ nav:
           - URL Reader: 'plugins/url-reader.md'
       - Testing:
           - Testing with Jest: 'plugins/testing.md'
+          - Dry Run Testing: 'plugins/dry-run-testing.md'
```

Comment:
You'll also want to add this here:

https://github.com/backstage/backstage/blob/e43d17bea734b6cb271419093c6a0865376c186c/microsite/sidebars.json#L109

This is the file that drives the sidebar found here: https://backstage.io/docs